eng/Generation.psm1

```diff
@@ -50,7 +50,7 @@ function AutoRest-Reset()
     Invoke "$script:autoRestBinary --reset"
 }
 
-function Invoke-Cadl($baseOutput, $projectName, $mainFile, $sharedSource="", $fast="", $debug="")
+function Invoke-Cadl($baseOutput, $projectName, $mainFile, $arguments="", $sharedSource="", $fast="", $debug="")
```

Comment:
@archerzz It may be many parameters which will be used in cadl comple command, something like the command parameter to call autorest. it is not the parameter for this function. So I think we just provide such function parameter '$arguments' for custom to provide options/argument to call an command.

Reply:
> @archerzz It may be many parameters which will be used in cadl comple command, something like the command parameter to call autorest. it is not the parameter for this function. So I think we just provide such function parameter '$arguments' for custom to provide options/argument to call an command.

I see. Then I suggest to move this to the end.